test/evp_extra_test.c

```diff
@@ -2530,10 +2545,22 @@ static int test_evp_iv(int idx)
             || !TEST_mem_eq(ref_iv, ref_len, iv, ivlen))
         goto err;
 
+    /* For CBC cipher test that the updated iv is reset after reinit */
```

Comment:
This comment still only talks about CBC and not the other modes (but maybe this is not critical to have correct)

Reply:
Fixed